docker-container/install.sh

```diff
@@ -0,0 +1,14 @@
+# Initialize Magento2 Enviroment
+set -x
+source config.sh
+currentPhpVersion=$(php -v | head -n 1 | cut -d " " -f 2)
+if [[ $currentPhpVersion != *"7.0"* &&  $currentPhpVersion != *"7.1"*  &&  $currentPhpVersion != *"7.2"* ]]; then
+    brew update
+    brew install php@7.2
+    echo 'export PATH="/usr/local/opt/php@7.2/bin:$PATH"' >> ~/.bash_profile
+    source ~/.bash_profile
+fi
+rm -rf ../../magento-cloud/
+git clone -b $m2Version https://github.com/magento/magento-cloud.git ../../magento-cloud
+python auth-format.py $magentoCloudPublicKey $magentoCloudPrivateKey
+cp auth.json ../../magento-cloud/
```

Comment:
For development process is very important to enable developer mode for Magento, of course after Magento was installed.

` php bin/magento deploy:mode:set developer`

Reply:
@ethanwayda22 i am not sure that i inserted my comment above to the correct step, if you think it was not a correct place, please move to the correct place.

Reply:
@ethanwayda22 Can you insert it in docker-container/configure-store.sh ?
After you created an admin user.

Reply:
@YevhenShBolt I have attempted to add developer mode but it seems to behave very slowly when we do. I followed the directions here (https://devdocs.magento.com/guides/v2.3/cloud/docker/docker-config.html) which were recently updated. I think it is best to save this for a further update and begin testing with the current version.